.mergify.yml

```
@@ -0,0 +1,105 @@
+      - check-skipped = gateway-contracts-integrity-checks/contract-integrity-checks (bpr)
+      - check-success = gateway-contracts-deployment-tests/sc-deploy (bpr)
+      - check-neutral = gateway-contracts-deployment-tests/sc-deploy (bpr)
+      - check-skipped = gateway-contracts-deployment-tests/sc-deploy (bpr)
+      - check-success = gateway-contracts-hardhat-tests/tests (bpr)
+      - check-neutral = gateway-contracts-hardhat-tests/tests (bpr)
+      - check-skipped = gateway-contracts-hardhat-tests/tests (bpr)
+      - check-success = gateway-contracts-docker-build/docker-hardhat-test (bpr)
+      - check-neutral = gateway-contracts-docker-build/docker-hardhat-test (bpr)
+      - check-skipped = gateway-contracts-docker-build/docker-hardhat-test (bpr)
+      - check-success = gateway-contracts-mock-deployment/deploy (bpr)
+      - check-neutral = gateway-contracts-mock-deployment/deploy (bpr)
+      - check-skipped = gateway-contracts-mock-deployment/deploy (bpr)
+      - check-success = gateway-contracts-hardhat-tests-docker/test (bpr)
+      - check-neutral = gateway-contracts-hardhat-tests-docker/test (bpr)
+      - check-skipped = gateway-contracts-hardhat-tests-docker/test (bpr)
+      - check-success = gateway-contracts-upgrade-tests/sc-upgrade (bpr)
+      - check-neutral = gateway-contracts-upgrade-tests/sc-upgrade (bpr)
+      - check-skipped = gateway-contracts-upgrade-tests/sc-upgrade (bpr)
+      - check-success = kms-connector-tests/test-connector (bpr)
+      - check-neutral = kms-connector-tests/test-connector (bpr)
+      - check-skipped = kms-connector-tests/test-connector (bpr)
+      - check-success = kms-connector-db-migration-docker-build/build (bpr)
+      - check-neutral = kms-connector-db-migration-docker-build/build (bpr)
+      - check-skipped = kms-connector-db-migration-docker-build/build (bpr)
+      - check-success = kms-connector-dependency-analysis/dependencies-check (bpr)
+      - check-neutral = kms-connector-dependency-analysis/dependencies-check (bpr)
+      - check-skipped = kms-connector-dependency-analysis/dependencies-check (bpr)
+      - check-success = sdk-rust-sdk-tests/test-rust-sdk (bpr)
+      - check-neutral = sdk-rust-sdk-tests/test-rust-sdk (bpr)
+      - check-skipped = sdk-rust-sdk-tests/test-rust-sdk (bpr)
+      - check-success = library-solidity-hardhat-tests/build (bpr)
+      - check-neutral = library-solidity-hardhat-tests/build (bpr)
+      - check-skipped = library-solidity-hardhat-tests/build (bpr)
+      - check-success = test-suite-docker-build/docker-e2e-image (bpr)
+      - check-neutral = test-suite-docker-build/docker-e2e-image (bpr)
+      - check-skipped = test-suite-docker-build/docker-e2e-image (bpr)
+pull_request_rules:
+  - name: autoqueue-to-main
+    description: Queue a PR for main when it matches the queue conditions
+    conditions:
+      - base = main
+      - "#approved-reviews-by >= 1"
+      - label=ready-to-merge
+      - label!=do-not-merge
```

```suggestion
      - label=ready-to-merge
```

nitpick: you can remove those 3 conditions as they are already in your `queue_conditions`.